src/app/main-page/components/categories/categories.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'food-categories',
+  templateUrl: './categories.component.html',
+  styleUrls: ['./categories.component.scss'],
+})
+export class CategoriesComponent implements OnInit {
+  CATEGORIES = [
```

Comment:
Why do we need to keep it inside a component? why not import from somewhere like a constant?